ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -72,6 +72,7 @@
   private final List<NetworkSend> completedSends;
   private final List<NetworkReceive> completedReceives;
   private final List<String> disconnected;
+  private final List<String> interimDisconnectedList;
```

Comment:
don't think this is a very descriptive name. Something along the lines of `initiatedDisconnects` or `closedConnections` would be better.